### ISSUE TITLE
Page component - Nav is visibly hidden but not hidden from AT

### ISSUE THREAD

**Issue by jgiardino** [patternfly/patternfly-next#1421]
Currently, when the page navigation is hidden using CSS, it isn't hidden from assistive technology like screen readers.

Whenever CSS is used to hide elements (instead of using the `hidden` attribute on the HTML element), the following properties should be defined to ensure that the contents are completely hidden from screen readers:
```
visibility: hidden; 
display: none;
```
